parsec/core/gui/menu_widget.py

```diff
@@ -28,35 +29,35 @@ def __init__(self, *args, **kwargs):
         self.icon_connection.apply_style()
         self.button_enrollment.hide()
 
-    def paintEvent(self, _):
+    def paintEvent(self, _: object) -> None:
```

Comment:
Should be a `QPaintEvent` (https://doc.qt.io/qt-5/qpaintevent.html). Not really important since it isn't used by might as well type it correctly.

Reply:
Done.